scripts/sanity_chk/sanitylib.py

```diff
@@ -2212,6 +2212,8 @@ def __init__(self, board_root_list=[], testcase_roots=[], outdir=None):
         self.start_time = 0
         self.duration = 0
         self.warnings = 0
+        self.SAMPLE_FILENAME = 'sample.yaml'
```

Comment:
class variables, not instance variables would be better.

Reply:
fixed